libraries/AP_GPS/AP_GPS_NOVA.cpp

```diff
@@ -226,7 +226,7 @@ AP_GPS_NOVA::process_message(void)
                 case 34: // narrow float
                 case 48: // l1 int
                 case 50: // narrow int
-                    state.status = AP_GPS::GPS_OK_FIX_3D_RTK;
+                    state.status = AP_GPS::GPS_OK_FIX_3D_RTK_FLOAT;
```

Comment:
I hadn't looked up documentation at the time :) Was hoping someone would tell me what the split is supposed to be, but I agree the ones you proposed seem reasonable from the comments.